docs/source/tutorials.rst

```diff
@@ -15,40 +15,217 @@ Prerequisites
+We need to add columns for the FASTA and GFF file paths and reorder them. Run the following commands:
 
 .. code-block:: console
 
-    conda activate microGWAS
+   We need a code to enable the user to modify their phentoype file to match the printed output. Perhaps some few bash commands?
+Verify the updated phenotype file:
+
+.. code-block:: console
+
+   head -n 5 data/updated_data.tsv
+
+You should see output similar to this:
+
+.. code-block:: none
+
+   strain  fasta   gff     killed  phenotype
+   ECOR-01 data/fastas/ECOR-01.fasta       data/gffs/ECOR-01.gff   0       0
+   ECOR-02 data/fastas/ECOR-02.fasta       data/gffs/ECOR-02.gff   10      1
+   ECOR-03 data/fastas/ECOR-03.fasta       data/gffs/ECOR-03.gff   0       0
+   ECOR-04 data/fastas/ECOR-04.fasta       data/gffs/ECOR-04.gff   0       0   
+
+5. Set up the environment and configure the pipeline:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+a. Set up the eggnog-mapper database:
+
+   If you have already downloaded the ``eggnog-db``:
+   
+   Create a symbolic link for the directory where the eggnog-mapper databases are located:
+
+   .. code-block:: console
+
+      ln -s /fast-storage/miniconda3/envs/eggnog-mapper/lib/python3.9/site-packages/data/ data/eggnog-mapper
+
+   If you have not downloaded the ``eggnog-db``:
+   
+   Run the following command to download the eggnog-db:
+
+   .. code-block:: console
+
+      snakemake -p data/eggnog-mapper/eggnog.db --cores 8 --use-conda --conda-frontend mamba
+
+   After the download is complete, create the symbolic link as shown above.
+
+b. Configure the pipeline:
+
+   Edit the ``config/config.yaml`` file. In the ``##### params #####`` section, uncomment and adjust the following parameters:
+
+   .. code-block:: yaml
+
+      targets: [
+         "phenotype",
+         #"phenotype2",
+      ]
+      
+      # MLST scheme
+      mlst_scheme: ecoli
+
+      # references for association summaries and annotation
+      summary_references: "--reference 536 --reference CFT073 --reference ED1a --reference IAI1 --reference IAI39 --reference K-12_substr._MG1655 --reference UMN026 --reference UTI89"
+      annotation_references: "--focus-strain 536 --focus-strain CFT073 --focus-strain ED1a --focus-strain IAI1 --focus-strain IAI39 --focus-strain K-12_substr._MG1655 --focus-strain UMN026 --focus-strain UTI89"
+      enrichment_reference: "IAI39"
+      
+      # species to be used for AMR and virulence predictions
+      species_amr: "Escherichia"
+
+   Note: These parameters are initially commented out in the file. You need to uncomment them (remove the '#' at the start of each line) and adjust as necessary for your analysis.
+
+Running the microGWAS pipeline
+------------------------------
 
 Run the bootsrapping script.
 
 .. code-block:: console
 
-    bash bootstrap.sh 536.fasta,CFT073.fasta,ED1a.fasta,IAI1.fasta,IAI39.fasta,K-12_substr._MG1655.fasta,UMN026.fasta,UTI89.fasta
+    bash bootstrap.sh Escherichia coli IAI39 
```

Comment:
This would still not work, see the README for a working command

Reply:
Sorry, forgot to modify this bit.